lib/LocalDeepnet.js

```diff
@@ -516,8 +516,9 @@ LocalDeepnet.prototype.deepnetPredict = function (inputData) {
   inputArray = this.fillArray(inputData, uniqueTerms);
   if (typeof this.networks !== 'undefined') {
     return this.predictList(inputArray);
+  } else {
```

Comment:
don't need to change it now, but when there's an if and you return something inside, you don't need else.. I don't know if it's a matter or style but we usually prefer to avoid extra code